packages/main/src/Avatar.ts

```diff
@@ -170,6 +184,12 @@ class Avatar extends UI5Element implements ITabbable {
 	@property({ type: AvatarSize, defaultValue: AvatarSize.S })
 	_size!: AvatarSize;
 
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
```

Comment:
You don't need this as a metadata property, that invalidates.

Use getter instead:
`get _interactive() { return this.interactive && !this.disabled;}`

Remove:
`@property({ type: Boolean }) _interactive!: boolean;`

Remove this from onBeforeRendering:
`this._interactive = this.interactive && !this.disabled;`